lib/spotify_api.rb

```diff
@@ -55,10 +55,92 @@ def find_song_spotify song
         :album_name =>  song["album"]["name"],
         :album_image => song["album"]["images"][2],
         :artist => song["artists"][0]["name"],
-        :preview_url => song["preview_url"]
+        :preview_url => song["preview_url"],
+        :uri => song["uri"]
       }
```

Comment:
A thought for later: it's a little obnoxious that this `uri` problem happened at all. Would it make sense for this method to return an array of (not-yet-saved) `Song` objects?